### PR TITLE
Changed header for import data status for datafile and printing size …

### PR DESCRIPTION
**Pull request by ShivanshGahlot**
…in megabytes instead of bytes

### CODE REVIEW

File: yb-voyager/cmd/importDataStatusCommand.go [hunk 1]
```diff
@@ -33,8 +33,8 @@ func init() {
 type tableMigStatusOutputRow struct {
 	tableName          string
 	status             string
-	totalRowCount      int64
-	importedRowCount   int64
+	totalRowCount      float64
+	importedRowCount   float64
```
Comment (amit-yb) [inline yb-voyager/cmd/importDataStatusCommand.go:37]: Keep it `int64`.
Rename these two vars as `totalCount` and `importedCount`. Add a comment stating, the vars hold "row count" for `import data` and "byte count" for `import data file`.

File: yb-voyager/cmd/importDataStatusCommand.go [hunk 2]
```diff
@@ -91,10 +97,22 @@ func runImportDataStatusCmd() error {
 		row := &tableMigStatusOutputRow{
 			tableName:          tableName,
 			status:             status,
-			totalRowCount:      totalRowCount,
-			importedRowCount:   importedRowCount,
+			totalRowCount:      float64(totalRowCount),
+			importedRowCount:   float64(importedRowCount),
```
Comment (amit-yb) [inline yb-voyager/cmd/importDataStatusCommand.go:101]: Similarly rename the local variables as `totalCount` and `importedCount`.

File: yb-voyager/cmd/importDataStatusCommand.go [hunk 3]
```diff
@@ -91,10 +97,22 @@ func runImportDataStatusCmd() error {
...
+
+		if dataFileDescriptor.TableRowCount == nil {
+			// case of importDataFileCommand where file size is available not row counts
+
+			// converting size in bytes to megabytes
+			totalRowSize := float64(totalRowCount) / 1000000.0
+			importedRowSize := float64(importedRowCount) / 1000000.0
+
+			row.totalRowCount = totalRowSize
+			row.importedRowCount = importedRowSize
```
Comment (amit-yb) [inline yb-voyager/cmd/importDataStatusCommand.go:113]: Perform this calculation right in the `Printf()` call.

**Review by amit-yb — COMMENTED** [review]
Just a few minor nits.

File: yb-voyager/cmd/importDataStatusCommand.go (hunk 1 above)
Comment (ShivanshGahlot) [reply yb-voyager/cmd/importDataStatusCommand.go]: Done

File: yb-voyager/cmd/importDataStatusCommand.go (hunk 3 above)
Comment (ShivanshGahlot) [reply yb-voyager/cmd/importDataStatusCommand.go]: Done all of the above changes

File: yb-voyager/cmd/importDataStatusCommand.go [hunk 4]
```diff
@@ -30,11 +30,13 @@ func init() {
 	importDataCmd.AddCommand(importDataStatusCmd)
 }
 
+//totalCount stores the total number of rows in case of import data command and the total size of the file in bytes in case of import data file command.
+//importedCount stores the number of rows imported in case of import data command and the size of the file imported in bytes in case of import data file command.
```
Comment (amit-yb) [inline yb-voyager/cmd/importDataStatusCommand.go:34]: Nit:
`totalCount` and `importedCount` store row-count for `import data` command and byte-count for `import data file` command.

**Review by amit-yb — APPROVED** [review]
LGTM with a minor nit. Thanks for this change.

**Pull request merged**